docs/sources/project/windows-client-test-server.md

```diff
@@ -0,0 +1,139 @@
+page_title: Set up a Windows server for client testing
+page_description: How to set up a server to test Docker Windows client
+page_keywords: development, inception, container, image Dockerfile, dependencies, Go, artifacts, windows
+
+
+# Windows Client test server
```

Comment:
also the filename, I'd recommend changing this to "Testing Windows Docker Client"